softdevice_controller/CHANGELOG.rst

```diff
@@ -18,12 +18,12 @@ Added
 =====
 
 * Added experimental support for Periodic Advertising.
-  Use :c:func:`sdc_support_le_periodic_adv` or :c:func:`sdc_support_le_periodic_sync` or both to enable this feature.
```

Comment:
I think this change is unintended. Probably I missed to backport this to dragoon.